apps/mobile/src/libs/components/screen-wrapper/screen-wrapper.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Edge, useSafeAreaInsets } from "react-native-safe-area-context";
+
+import { View } from "~/libs/components/components";
+import { globalStyles } from "~/libs/styles/styles";
+
+const ZERO_PADDING = 0;
+const DEFAULT_SAFE_AREA_EDGES: Edge[] = ["top", "bottom", "left", "right"];
+
+type Properties = {
+	children: React.ReactNode;
+	edges?: Edge[];
+};
+
+const ScreenWrapper: React.FC<Properties> = ({ children, edges }) => {
+	const insets = useSafeAreaInsets();
+
+	const localEdges = edges || DEFAULT_SAFE_AREA_EDGES;
+
+	const paddingStyle = {
+		paddingBottom: localEdges.includes("bottom") ? insets.bottom : ZERO_PADDING,
+		paddingLeft: localEdges.includes("left") ? insets.left : ZERO_PADDING,
+		paddingRight: localEdges.includes("right") ? insets.right : ZERO_PADDING,
+		paddingTop: localEdges.includes("top") ? insets.top : ZERO_PADDING,
+	};
```

Comment:
```suggestion
const ScreenWrapper: React.FC<Properties> = ({ children, edges = DEFAULT_SAFE_AREA_EDGES  }) => {
	const insets = useSafeAreaInsets();

	const paddingStyle = {
		paddingBottom: edges.includes("bottom") ? insets.bottom : ZERO_PADDING,
		paddingLeft: edges.includes("left") ? insets.left : ZERO_PADDING,
		paddingRight: edges.includes("right") ? insets.right : ZERO_PADDING,
		paddingTop: edges.includes("top") ? insets.top : ZERO_PADDING,
	};
```

Reply:
I try this approach and .includes method highlights red - 
![image](https://github.com/user-attachments/assets/1363f26f-a611-4c19-9d96-7c15019468c1)
Can I try add edges?.includes?